### ISSUE TITLE
Still alive? mingw32-make?

### ISSUE THREAD

**Issue by jjerome00** [Dirkster99/NextLevel#1]
I was really interested in your project.  I'm trying to generate some docs for my database, and I wanted to play around with your project more.  However, I cannot figure out mingw.  I downloaded the library, but I cannot use the bat file you provide to call "mingw32-make".  I know enough c++ to be dangerous,  but I'm clueless as far as using this tool.  Are there any docs that can hold my hand as far as building this project?

**Comment by Dirkster99** [comment]
You can also build this with Visual Studio, have a look here:
https://github.com/Dirkster99/NextLevel/blob/master/DevProj/MSDev/MSDev.sln

You could also try defining a new makefile or Visual Studio solution - it should not be too difficult. I only started out with mingw because Visual Studio was not available back then. Let me know how you go I would also be interested in developing this further - in the parser section in particular since Microsoft has opened up some DACPac API*s around SQLServer ...

**Comment by jjerome00** [comment]
That did the trick - thanks!  I'm leaning towards rolling a solution using MS's system views, which seem to be able to do everything these days, including dependencies.

**Comment by Dirkster99** [comment]
yes same here, I am also leaning towards the same end. Which is why the source code is available for inspiration and those who need a solution now. But its not really developed anymore. I would implement it these days in Java or .Net and make use of an API like the DacPac API from Microsoft.

My problem is just that I have not found a working solution, yet. This is still under heavy development as we speak, right? Please let me know if you have interesting pointers on this subject. Thanks

**Comment by jjerome00** [comment]
I've never heard of DacPac before, I had to look it up.  I used SMO to build a little database object scripter app (c#) for use at a client - mostly to standardize our scripts for use with subversion.

What I'm doing now is trying to satisfy an itch for creating database documentation - preferably in a wiki format.  I've seen some tools that come close, but they all seem to fall short in one way or another.
